src/commitment_scheme/kzg10/srs.rs

```diff
@@ -72,10 +72,17 @@ impl PublicParameters {
         })
     }
 
+    /// Serialises a [`PublicParameters`] struct into a slice of bytes
+    pub fn to_bytes(&self) -> Vec<u8> {
```

Comment:
This generates a `Vec` and should be `into_bytes`

Reply:
True. But we need to take into account that this will break API. For me it's fine. I'll refactor it

Reply:
I think its better we do it now because it will later on conflict with the traits that are being implemented by @ZER0 if we leave it